opflexagent/vpp_plug_firewall.py

```diff
@@ -28,8 +28,7 @@ def vpp_nova_to_osvif_vif_vhostuser(vif):
         except NotImplementedError as e:
             if vif['details'].get("vhostuser_vpp_plug", False):
                 profile = objects.vif.VIFPortProfileOpenVSwitch(
-                    interface_id=vif.get('ovs_interfaceid') or vif['id'],
-                    datapath_type=model.VIF_DETAILS_OVS_DATAPATH_NETDEV)
+                    interface_id=vif.get('ovs_interfaceid') or vif['id'])
```

Comment:
not that familiar with the code but why we don't need to pass in the datapath_type parameter now?

Reply:
This function derives from nova/nova/network/os_vif_util.py _nova_to_osvif_vif_vhostuser. This argument was added post ocata. I had to make this change for testing in ocata. For the master branch, I can have this argument still, I will add it back. Thanks.